### ISSUE TITLE
Remove References to Staticman Public Instances in Docs

### ISSUE THREAD

**Issue by ghost** [eduardoboucas/staticman#347]
According to https://github.com/eduardoboucas/staticman/issues/317#issuecomment-565250060, the public instance for staticman is no longer supported.

In the [docs](https://staticman.net/docs/), there are still references on how to setup staticman using the public instance.

I believe these references should be removed.

**Comment by alexwaibel** [comment]
You are correct. See #343 for some additional details about the docs. It's on my todo list to fix that but I haven't had time to get to it yet. In the meantime, anyone is welcome to submit a PR on the documentation repo if so inclined.
